Having this error when building ros2_mara_ws (see [MARA repo](https://github.com/AcutronicRobotics/MARA#compile-the-ros-20-workspace)) using your fork as `robotiq_modular_gripper`

`/home/developer/ros2_mara_ws/src/robotiq_modular_gripper/robotiq_gripper_gazebo_plugins/src/spline.cpp:1:10: fatal error: mara_gazebo_plugins/spline.hpp: No such file or directory
 #include "mara_gazebo_plugins/spline.hpp"
`

I'm having a look

--- a/robotiq_gripper_gazebo_plugins/src/spline.cpp
+++ b/robotiq_gripper_gazebo_plugins/src/spline.cpp
@@ -1,4 +1,4 @@
-#include "mara_gazebo_plugins/spline.hpp"
+#include "robotiq_gripper_gazebo_plugins/spline.hpp"
 namespace tk
 {
   // band_matrix implementation

